src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -149,6 +149,36 @@ NonlinearSolverParameters::NonlinearSolverParameters( string const & name,
     setApplyDefaultValue( 0 ).
     setDescription( "Flag to decide whether to iterate between sequentially coupled solvers or not." );
 
+  registerWrapper( viewKeysStruct::useDBCString(), &m_useDBC ).
```

Comment:
Minor concern but I think there is a recommendation to use full length name for readability in GEOS.

Reply:
let's start with "GEOS" then haha